.github/workflows/test.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        node-version: [10.x, 20.x]
+        node-version: [12.x, 20.x]
```

Comment:
We still need to run test on Node.js@10, please use `if` and use old jest version there (for Node.js@10), in package json use the lastest version, i.e. 28

Reply:
The package.json version is already up-to-date, the main purpose of this PR was to increase the minimum Jest version across all CI because Jest 27 doesn't support failing tests (i.e. `it.failing`).

Unfortunately, Jest 28 dropped support Node v10. It was mentioned https://github.com/webpack/webpack/pull/18572#discussion_r1677173885 that Node v10 support should be dropped from webpack, so I figured upgrading the minimum CI test suite to v12 should be uncontroversial.